.github/workflows/lighthouse-check.yml

```diff
@@ -0,0 +1,16 @@
+name: Lighthouse CI
+on: push
+jobs:
+  lighthouse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Audit URLs using Lighthouse
+        uses: treosh/lighthouse-ci-action@v3
+        with:
+          urls: |
+            https://open-discourse-ui-git-develop.ofranke.vercel.app/
+            https://open-discourse-ui-git-develop.ofranke.vercel.app/suche
```

Comment:
`/suche` oder `/tools-und-daten`? Dachte es gäbe kein `/suche` mehr?

Reply:
oh, stimmt. ja die ganzen urls würde ich da sowieso noch richtig reinsetzen, wenn alles ready ist. bis dahin können wir das PR ja offen lassen